app/assets/stylesheets/application.scss

```diff
@@ -48,6 +71,15 @@ footer {
   font-size: 16px;
 }
 
+.name img {
+  background: inherit;
+  margin: 10px;
```

Comment:
Properties should be ordered background, height, margin, padding-bottom, padding-left, width